### PR TITLE
Update website pages

### PR DESCRIPTION
**Pull request by Rachel1418**

### CODE REVIEW

**Comment by AndrewLester** [comment]
Very nice work on this!

File: _pages/robot/2020.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <ul>
+    <li><strong>Vision Software:</strong> With vision tape on the tower, having good and accurate vision code is very important, especially since we have to shoot balls a great distance. Programmed in Python, this software uses data from a camera located right under the shooter and lots of math to position the robot correctly. There is also a button on one of the joysticks that runs software that aligns the robot to the vision tape on the tower.
+    </ul>
+    <p>As usual, our code is open source, and can be found on <a href="https://www.github.com/frc1418">our GitHub organization</a>. Here are some quick links:</p>
+    <ul>
+      <!--TODO: change the website code here-->
+        <li><a href="https://github.com/frc1418/2020-robot">Robot code</a></li>
+        <li><a href="https://github.com/frc1418/2020-dashboard">Dashboard code</a></li>
+        <li><a href="https://github.com/frc1418/2020-vision">Vision code</a></li>
+        <li><a href="https://github.com/frc1418/VictiScout">VictiScout scouting system (NEEDS TO BE UPDATED)</a></li>
```
Comment (ErikBoesen) [inline _pages/robot/2020.html]: Does it?

Comment (ErikBoesen) [reply _pages/robot/2020.html]: I think we should not publish the page with this visible

File: _pages/robots.html [hunk 2]
```diff
@@ -11,10 +11,15 @@
     <!-- TODO: Expand this page. There's not enough content. Also, would be nice to have a more interesting layout. maybe even have pixel drawings of the robots? -->
     <h1>The Robots</h1>
     <div>
+        <div>
+            <img src="/images/robots/2020.png">
+            <h5>2020</h5>
+            <p>Our 2020 robot is being build for the <a href="https://www.youtube.com/watch?v=gmiYWTmFRVE"> FIRST INFINITE RECHARGE competition</a>. It has an intake system that can take up powercells (softballs) and shoot them at varying distances, by using a motor to spin wheels very fast, into a tall tower. It also has a wheel that extends by a piston that is used to spin the control panel (color wheel) with various sensors to detect the color and the closeness of the structure to the robot. Finally, we have installed a scissor lift that extends to hook onto the seesaw and a winch that is used for lifting the robot off the ground during matches. <a href="/robot/2020">More about our 2020 robot</a></p>
```
Comment (ErikBoesen) [inline _pages/robots.html]: "being build" -> "being built"

Comment (ErikBoesen) [reply _pages/robots.html]: Also, no space at the start of the link.

Comment (ErikBoesen) [reply _pages/robots.html]: Also they're not softballs are they? I thought they were gator balls or whatever those are called.

File: _pages/robots.html [hunk 3]
```diff
@@ -11,10 +11,15 @@
...
+        </div>
         <div>
             <img src="/images/robots/2019.png">
             <h5>2019</h5>
-            <p> Our 2019 robot is being built for the <a href="https://www.youtube.com/embed/Mew6G_og-PI"> FIRST "Destination: Deep Space" competition</a>. It has an extentable elevator with components that could handle both the hatches and cargo of the challenge with ease. It also has multiple pistons on the bottom to push the robot onto the second HAB level at the end of the game. <a href="/robot/2019">More about our 2019 robot</a></p>
+            <p> Our 2019 robot, Habibti, was built for the <a href="https://www.youtube.com/embed/Mew6G_og-PI"> FIRST "Destination: Deep Space" competition</a>. It has an extentable elevator with components that could handle both the hatches and cargo of the challenge with ease. It also has multiple pistons on the bottom to push the robot onto the second HAB level at the end of the game. <a href="/robot/2019">More about our 2019 robot</a></p>
```
Comment (ErikBoesen) [inline _pages/robots.html]: I fully endorse this change 😤 if possible let's add other names. This page has some names for our very early robots that you can draw from! https://firstwiki.github.io/frc1000/1418

File: _pages/team.html [hunk 4]
```diff
@@ -43,7 +43,7 @@ <h5>Jack Evans</h5>
             <img src="/images/team/andrew-lester.png">
             <p class="label">🖥️ Programmer 🖥️</p>
             <h5>Andrew Lester</h5>
-            <p>Andrew is a programmer with years of experience in Java, Python, and HTML/CSS. He will handle our foundational robot code during the 2019 season and has worked on tasks from updating our website to maintaining our open-source projects.</p>
+            <p>Andrew is a programmer with years of experience in Java, Python, and HTML/CSS. He will handled our foundational robot code during the 2019 season and has worked on tasks from updating our website to maintaining our open-source projects.</p>
```
Comment (ErikBoesen) [inline _pages/team.html]: Why?

File: _pages/team.html [hunk 5]
```diff
@@ -55,7 +55,7 @@ <h5>Joe Carpenter</h5>
             <img src="/images/team/stephanie-synnott.png">
             <p class="label">⛑️ Safety Captain ⛑️</p>
             <h5>Stephanie Synnott</h5>
-            <p>Stephanie is 1418's Safety Captain, and additionally works on outreach.</p>
+            <p>Stephanie is our Safety Captain. She strives to be an approachable mentor for newer Safety Captains, and has received the second place Safety Award at the 2019 Oxon Hill event. Furthermore, she has been voted "Star of the Day" numerous times, including at the 2019 Cheaspeake District event. She also works with outreach to plan many of our fundrasing events.</p>
```
Comment (ErikBoesen) [inline _pages/team.html]: Great stuff!

File: _pages/team.html [hunk 6]
```diff
@@ -152,7 +152,104 @@ <h5>Ricardo Quiroz </h5>
...
+            <p class="label">🖥️ Programmer 🖥️</p>
+            <h5>Smridhi Madan</h5>
+            <p>Smridhi is a new member of the team and is a programmer.</p>
+        </div>
+
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">🔧 Mechanic 🔧</p>
+            <h5>Will Patmore</h5>
+            <p>(Description in progress)</p>
```
Comment (ErikBoesen) [inline _pages/team.html]: Hmm, can we just add a basic "Will is a mechanic" description like the other people have? This looks a tad unprofessional

File: _pages/team.html [hunk 7]
```diff
@@ -152,7 +152,104 @@ <h5>Ricardo Quiroz </h5>
...
+        </div>
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">🔧 Mechanic 🔧</p>
+            <h5>Jacob Philidelphia</h5>
```
Comment (ErikBoesen) [inline _pages/team.html]: Are you sure this name is spelled right?

File: _pages/team.html [hunk 8]
```diff
@@ -152,7 +152,104 @@ <h5>Ricardo Quiroz </h5>
...
+            <img src="/images/team/placeholder.png">
+            <p class="label">🔧 Mechanic 🔧</p>
+            <h5>Sophia Urbom</h5>
+            <p>(Description in progress)</p>
+        </div>
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">🖥️ Programmer 🖥️</p>
+            <h5>Mena Hailemariam</h5>
+            <p>(Description in progress)</p>
```
Comment (ErikBoesen) [inline _pages/team.html]: Same sentiment about all these descriptions.

File: _pages/team.html (hunk 4 above)
Comment (AndrewLester) [reply _pages/team.html]: I think this is supposed to be: "he handled"

**Pull request merged**